### ISSUE TITLE
Voice chat on cs16cli doesnt work

### ISSUE THREAD

**Issue by lostfxo** [FWGS/xash3d-fwgs#1390]
loopback works but voice chat between clients doesnt really work

**Comment by a1batross** [comment]
Just because you reposted it from Discord to GitHub won't really help. Well, at least it's easier to track there.

**Comment by lostfxo** [comment]
I reposted because Velaron just said "Leave an issue on github if it doesn't exist"

**Comment by Velaron** [comment]
Yeah but it should have been on cs16-client repo, as it seems to be that project's specific issue

**Comment by lostfxo** [comment]
Yeah but as it seems it doesnt really work on other mods tho
